src/components/NcListItem/NcListItem.vue

```diff
@@ -31,6 +31,7 @@
 			:active="true"
 			:details="'1h'"
 			:counter-number="44"
+			one-line
```

Comment:
I'm not sure it should be in the default usage example. It could be confusing mixing with the default layout. I'd show it in a separated example in the docs.

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/74e5b060-bc92-4b5d-869b-3654c13ad8bc)